client-misk-spanner/src/main/kotlin/app/cash/backfila/client/misk/spanner/SpannerBackfill.kt

```diff
@@ -0,0 +1,20 @@
+package app.cash.backfila.client.misk.spanner
+
+import app.cash.backfila.client.Backfill
+import app.cash.backfila.client.BackfillConfig
+import com.google.cloud.spanner.DatabaseClient
+import com.google.cloud.spanner.KeyRange
+
+abstract class SpannerBackfill<Param : Any> : Backfill {
+  abstract val dbClient: DatabaseClient
+  abstract val tableName: String
+  abstract val primaryKeyColumns: List<String>
```

Comment:
Nit: I really like when we add documentation to these fields. It makes it much better in the future.

Also, I did realize recently that I think we could derive the primary key columns from the table name. IDK if that matters to you or not.

Reply:
Can you elaborate on how deriving the primary key columns would work? Would it require reading from the spanner schema information tables or something?

Reply:
I think you're talking about https://cloud.google.com/spanner/docs/information-schema#index_columns, and that feels like more effort than I really want right now.